Dockerfiles/midstream/README.md

```diff
@@ -15,18 +15,18 @@ First thing you reckon is that you are running this command directly from upstre
 
 ```
 git clone https://github.com/redhat-operator-ecosystem/operator-test-playbooks.git
-cd Dockerfiles
-### podman build -t <image_name>:<tagname> -f <filepath> --build-arg OPERATOR_SDK_VERSION=<operator_sdk_version> --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=<operator_test_playbooks_tag>
+cd operator-test-playbooks
+### podman build -t <image_name>:<tagname> -f <filepath> 
 ### example:
-podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0 --build-arg OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11
```

Comment:
Same question as above

Reply:
--build-arg OPERATOR_TEST_PLAYBOOKS_TAG=v1.0.11 is already present in Dockerfile